spec/system/jobseekers_can_subscribe_to_a_job_alert_spec.rb

```diff
@@ -135,8 +135,20 @@
     end
 
     context "when a location category search is carried out" do
+      let(:boundary_points) { [1.004562496029994090, 51.50833566307333, 1.005710530794815389, 51.5051084208278] }
```

Comment:
What's the purpose of this?

Reply:
You know what, I don't know. I think it may have been some attempt to not rely on anything else outside of what's being tested, but I assume that's unnecessary for a helper?

Reply:
Before adding buffer_coordinates and buffer_hash, when this test got to the stage where buffer suggestions were being built, it would fail as `LocationPolygon.with_name(location).buffers` was nil:

```
Failure/Error:
       buffer_vacancy_count = LocationPolygon.with_name(location).buffers.map do |buffer|
         [buffer.first, Search::AlgoliaSearchRequest.new(search_params.merge(polygon: [buffer.last])).stats.last]
       end

     NoMethodError:
       undefined method `map' for nil:NilClass
```

I also came across another failure which was caused by boundary being nil too, but I can't seem to recreate this, so I've removed `let(:boundary_points) { [1.004562496029994090, 51.50833566307333, 1.005710530794815389, 51.5051084208278] }`

Reply:
I would just stub that call (you do something similar in a different spec), since we don't care/ are not testing that it happens